modules/dreamview/frontend/dist/fonts/droid/README.txt

```diff
@@ -3,9 +3,9 @@ Copyright (C) 2008 The Android Open Source Project
 Licensed under the Apache License, Version 2.0 (the "License");
 you may not use this file except in compliance with the License.
 You may obtain a copy of the License at
-
+  
      http://www.apache.org/licenses/LICENSE-2.0
-
+  
```

Comment:
I saw these redundant whitespaces every time. Is it an IDE issue or intended format?

Reply:
files under dist are automatically generated.....

Reply:
Got it. Thank you!